codegenerator/cli/templates/static/codegen/src/Benchmark.res

```diff
@@ -62,34 +62,62 @@ module Data = {
     })
   }
 
+  module UpdateEndOfBlockRangeScannedData = {
+    type t = {
+      chainId: int,
+      elapsedTimeMillis: int,
+    }
+
+    let schema = S.object(s => {
+      chainId: s.field("chainId", S.int),
+      elapsedTimeMillis: s.field("elapsedTimeMillis", S.int),
+    })
+  }
+
   type t = {
     blockRangeFetched: array<BlockRangeFetched.t>,
     eventProcessing: array<EventProcessing.t>,
+    updateEndOfBlockRangeScannedData: array<UpdateEndOfBlockRangeScannedData.t>,
     startTime: Js.Date.t,
     mutable latestTime: Js.Date.t,
   }
   let schema = S.object(s => {
     blockRangeFetched: s.field("blockRangeFetched", S.array(BlockRangeFetched.schema)),
     eventProcessing: s.field("eventProcessing", S.array(EventProcessing.schema)),
+    updateEndOfBlockRangeScannedData: s.field(
+      "updateEndOfBlockRangeScannedData",
+      S.array(UpdateEndOfBlockRangeScannedData.schema),
```

Comment:
Maybe `S.schema` will be more convinient here DX-wise, since it doesn't require duplicating the field name.

Reply:
Can you link a doc or share an example? I'm not sure I understand exactly.

Reply:
https://github.com/DZakh/rescript-schema/blob/main/docs/rescript-usage.md#schema

Reply:
Ah nice I understand  👍🏼 yeah think that's better.